tabulator/__init__.py

```diff
@@ -3,10 +3,22 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+import io
+import os
+
+
+# General
 
 from .stream import Stream
 from . import exceptions
 
 # Deprecated
+
 from .topen import topen
 from .stream import Stream as Table
+
+# Version
+
+__version__ = io.open(
```

Comment:
`__init__.py` is definitely part of the package and installed in site-packages. Most libraries I know of define their version on `__init__.py` (eg https://github.com/kennethreitz/requests/blob/master/requests/__init__.py#L44 https://github.com/celery/celery/blob/master/celery/__init__.py#L17)